### PR TITLE
Renaming stuff

### PR DESCRIPTION
**Pull request by MarconZet**

### CODE REVIEW

**Comment by MarconZet** [comment]
Add some package structure under `org.virtuslab.akkasaferserializer`

**Comment by MarconZet** [comment]
@PawelLipski I'm not convinced about adding additional package for each subproject. The are already separated by longly named subfolders. This additional package would be redundant and wouldn't help anyone in any way.

**Comment by PawelLipski** [comment]
Okay, let's leave as is then, class names are also specific enough

File: akka-serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/AkkaSerializabilityCheckerCompilerPlugin.scala [hunk 1]
```diff
@@ -3,16 +3,16 @@ package org.virtuslab.akkasaferserializer
 import scala.tools.nsc.Global
 import scala.tools.nsc.plugins.{Plugin, PluginComponent}
 
-class AkkaSerializabilityCheckerPlugin(override val global: Global) extends Plugin {
+class AkkaSerializabilityCheckerCompilerPlugin(override val global: Global) extends Plugin {
```
Comment (PawelLipski) [inline akka-serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/AkkaSerializabilityCheckerCompilerPlugin.scala:6]: Hmmm everything in this project relates to Akka by default... e.g. there's just `dump schema event`, no mention of Akka.

Comment (PawelLipski) [reply akka-serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/AkkaSerializabilityCheckerCompilerPlugin.scala:6]: So I'd remove the `Akka` prefix unless you see a reason to leave it here specifically

File: build.sbt [hunk 2]
```diff
@@ -45,22 +45,22 @@ lazy val serializer = (projectMatrix in file("serializer"))
...
 
-lazy val codecs = (projectMatrix in file("codecs"))
+lazy val borerExtraCodecs = (projectMatrix in file("borer-extra-codecs"))
   .settings(name := "borer-extra-codecs")
   .settings(commonSettings)
   .settings(libraryDependencies ++= Seq(borerCore, akkaTyped % Provided, akkaStream % Provided, borerDerivation % Test))
   .jvmPlatform(scalaVersions = supportedScalaVersions)
 
-lazy val checkerLibrary = (projectMatrix in file("checker-library"))
+lazy val akkaSerializabilityCheckerLibrary = (projectMatrix in file("akka-serializability-checker-library"))
```
Comment (PawelLipski) [inline build.sbt:57]: Same for project/folder names, `akka-` prefix is probably redundant

File: dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/CrossVersionReporter.scala [hunk 3]
```diff
@@ -5,7 +5,7 @@ import scala.tools.nsc.reporters.Reporter
 
 /**
  * In update 2.12.12 -> 2.12.13 reporter was changed from method to value.
- * This code creates a reporter, regardless whether is a value of method
+ * This code creates a reporter, regardless whether it is a value of method
```
Comment (PawelLipski) [inline dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/CrossVersionReporter.scala:8]: `or method` ;)

File: serializability-checker-compiler-plugin/src/main/resources/scalac-plugin.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<plugin>
+    <name>akka-serializability-checker-plugin</name>
+    <classname>org.virtuslab.akkasaferserializer.SerializabilityCheckerCompilerPlugin</classname>
```
Comment (PawelLipski) [inline serializability-checker-compiler-plugin/src/main/resources/scalac-plugin.xml:3]: Okay so rename of the project/packages to ASH will happen on a follow-up PR right?

File: serializability-checker-compiler-plugin/src/test/scala/org/virtuslab/akkasaferserializer/compiler/AkkaSerializabilityCheckerCompiler.scala [hunk 5]
```diff
@@ -38,7 +38,7 @@ object AkkaSerializabilityCheckerCompiler {
     val compiler = new Global(settings, reporter) {
       override protected def computeInternalPhases(): Unit = {
         super.computeInternalPhases()
-        for (phase <- new AkkaSerializabilityCheckerCompilerPlugin(this).components)
```
Comment (PawelLipski) [inline serializability-checker-compiler-plugin/src/test/scala/org/virtuslab/akkasaferserializer/compiler/AkkaSerializabilityCheckerCompiler.scala]: There's still: `AkkaSerializabilityCheckerCompiler` and `AkkaSerializabilityCheckerCompilerSpec` to be renamed AFAICS

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**